demo/demo.html

```diff
@@ -1,48 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Bind input-date to date</title>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
-
-    <script>
-      window.Polymer = { lazyRegister: 'max', useNativeCSSProperties: !0, dom: 'shadow' };
-
-      // load webcomponents polyfills
-      (function() {
-        if ('registerElement' in document
-            && 'import' in document.createElement('link')
-            && 'content' in document.createElement('template')) {
+
+<head>
+  <title>Bind input-date to date</title>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <script src="/node_modules/web-animations-js/web-animations-next-lite.min.js"></script>
+  <script src="/node_modules/intl/dist/Intl.complete.js"></script>
+
+  <script>
+    window.Polymer = { lazyRegister: 'max', useNativeCSSProperties: !0, dom: 'shadow' };
+
+    // load webcomponents polyfills
+    (function () {
+      if ('registerElement' in document
+        && 'import' in document.createElement('link')
+        && 'content' in document.createElement('template')) {
```

Comment:
This needs to be updated as well. This is only valid for when using WC v0.

Use `web-components-loader` instead.